main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative './app'
+
+@app = App.new
+
+def list_options
+  puts '
+  Please choose one of the options: (1 - 7)
+    1. - List all books
+    2. - List all people
+    3. - Create a person
+    4. - Create a book
+    5. - Create a rental
+    6. - List all rentals for a given person id
+    7. Exit
+  '
+end
+
+def start_program
+  loop do
+    list_options
+    choice = gets.chomp.to_i
+    if choice.between?(1, 7)
+      if choice == 7
+        puts 'Welldone for using this app'
+        exit
+      else
+        choose_operation(choice)
+      end
+    end
+  end
+end
+
+def choose_operation(opt)
+  case opt
+  when 1
+    @app.list_books
+  when 2
+    @app.list_peoples
+  when 3
+    @app.create_person
+  when 4
+    @app.create_book
+  when 5
+    @app.create_rental
+  when 6
+    @app.list_rentals
+  end
+end
+
+def main
```

Comment:
- [ ] Your app doesn't seem to open kindly consider calling the main method at the end of the file so your app is displaying options on the screen
`main`

Reply:
Oh! Thanks. That was a mistake on my part. I will fix that now.